manifests/service.pp

```diff
@@ -13,7 +13,7 @@
     mode    => '0755',
   }
 
-  if $confluence::manage_service {
+  if str2bool($confluence::manage_service) {
     service { 'confluence':
```

Comment:
This should not be necessary with Puppet 4, though I suppose it doesn't particularly hurt, either.